e2e/_suites/ingest-manager/ingest-manager_test.go

```diff
@@ -75,6 +80,8 @@ func IngestManagerFeatureContext(s *godog.Suite) {
 				"minutes": minutesToBeHealthy,
 			}).Error("The Kibana instance could not get the healthy status")
 		}
+
+		imts.StandAlone.RuntimeDependenciesStartDate = time.Now()
```

Comment:
Marking when the runtime deps are started. We want to have here a lower boundary for queries